css/min-width-768px-style.css

```diff
@@ -39,6 +39,14 @@
   display: none;
 }
 
+.pop-menu {
```

Comment:
**_Kindly fix this issue:_**
_Your Menu should cover the `100%` of `height`:_
![Screen Shot 2021-12-13 at 17 16 56](https://user-images.githubusercontent.com/22258274/145904300-c27ef5a0-b4a0-419b-89e1-8df557ea2664.png)